configs/ACCIF435ELITE/config.h

```diff
@@ -0,0 +1,134 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU     AT32F435G
+
+#define BOARD_NAME        ACCIF435ELITE
+#define MANUFACTURER_ID   CUST
+
+#define USE_ACC
+#define USE_GYRO
+
+#define USE_GYRO_CLKIN
+#define USE_ACC_SPI_ICM42688P
+#define USE_GYRO_SPI_ICM42688P
+
+#define USE_BARO
+#define USE_BARO_DPS310
+#define DEFAULT_BARO_I2C_ADDRESS 119
+
+#define USE_FLASH
+#define USE_FLASH_M25P16
+#define USE_FLASH_W25Q128FV
+#define USE_FLASH_W25N01G
+
+// I/O
+#define LED0_PIN             PC15
+#define LED1_PIN             PC14
```

Comment:
Should not be used to drive LEDs

Reply:
Should be possible as it is? Sadly I have carried that from my previous design and I have already the boards with me and I was not planning to order more as is not cheap. Also the design of the PCB has been based on those pin position for the leds...
As far as I have been able to test on AT32 I have not faced any issue with the current provided with those pins for a long time already.

Reply:
Should keep an eye on https://betaflight.com/docs/development/manufacturer/manufacturer-design-guidelines 😉